src/GraphQLinq.Client/GraphContext.cs

```diff
@@ -8,15 +10,31 @@ namespace GraphQLinq
 {
     public class GraphContext
     {
-        protected GraphContext(string baseUrl, string authorization)
+        public HttpClient HttpClient
         {
-            BaseUrl = baseUrl;
-            Authorization = authorization;
+            get;
+        }
+
+        protected GraphContext(HttpClient httpClient)
```

Comment:
Is this ctor used anywhere?

Reply:
Not yet. It will be the main constructor in https://github.com/Giorgi/GraphQLinq/pull/26